tests/unit/scatter_test.coffee

```diff
@@ -0,0 +1,16 @@
+moduleForComponent 'scatter', '[Unit] Scatter component',
+  needs: [
+    'template:components/ember-chart'
+  ]
+
+test 'it renders', (assert) ->
```

Comment:
Even though they're in a component and have assertions, can we make these say "Scatter chart renders", etc instead of "it renders"